### PR TITLE
test/runtime: add stable image tags for memcache tests

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7027)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Review by nebril — APPROVED** [review]

**Comment by aanm** [comment]
test-me-please

File: test/runtime/memcache.go [hunk 1]
```diff
@@ -32,9 +32,9 @@ var _ = Describe("RuntimeMemcache", func() {
 	containers := func(mode string) {
 
 		images := map[string]string{
-			"memcache":             "memcached:1.5.11",
-			"memcache-bin-client":  "nebril/python-binary-memcached",
-			"memcache-text-client": "alpine",
+			"memcache":             "docker.io/library/memcached:1.5.11",
+			"memcache-bin-client":  "docker/cilium/python-bmemcached:v0.0.1",
```
Comment (ianvernon) [inline test/runtime/memcache.go]: this needs to be `docker.io` instead of `docker`

**Review by ianvernon — CHANGES_REQUESTED** [review]

**Comment by aanm** [comment]
test-me-please

**Review by vadorovsky — APPROVED** [review]

**Review by ianvernon — APPROVED** [review]

**Pull request merged**